### DISCUSSION TITLE
[Proposal] Allow dangling comma at the last property in record definition

### DISCUSSION THREAD

**Discussion by vini-nu** [dotnet/csharplang#5805] [Language Ideas] [answered]
![image](https://user-images.githubusercontent.com/6695244/154918105-8357269c-3b73-4e84-84b9-f312838465ef.png)

```csharp
using System;

namespace Some.Body.Once.Told.Me
{
    public record CurrentDataModel(
	    int SensorIdentifier,
	    double Pressure,
	    double Temperature,
	    DateTime DateTime,
	);
}
```

Allow comma at the last record property definition then defining records. We have similar in other areas in C#.

This will simply allow to use Ctrl+D in VisualStudio to copy-paste new property.

**Comment by Phantonia** [comment]
I think an argument can be made that more generally, trailing commas should be allowed in any parameter list, not just for records' primary constructor, but I do think that for consistency with most other comma seperated lists (object initializers, collection initializers, array initializers, `switch` expressions, etc.) this would be desirable.

**Comment by ufcpp** [comment, marked as answer]
https://github.com/dotnet/csharplang/issues/391 covers this.
